feat: enhance `Set-NsxtManagerAuthPolicy` for NSX 4.x password complexity (#260)
vmware-samples/power-validated-solutions-for-cloud-foundation: CHANGELOG.md

- Enhanced `Add-vRLIGroup` cmdlet with a vRealize Log Insight version check to throw a warning message if API is missing.
- Enhanced `Remove-vRLIGroup` cmdlet with a vRealize Log Insight version check to throw a warning message if API is missing.
- Enhanced `Add-vRSLCMNtpServer` cmdlet with additional checking of the inputs against the exiting configuration.
- Enhanced `Set-NsxtManagerAuthPolicy` to handle password complexity parameters for NSX 4.x.
- Enhanced `Get-LocalPasswordComplexity` to handle VMware Cloud Foudation version parameter.
- Enhanced `Get-LocalAccountLockout` to handle VMware Cloud Foudation version parameter.

> 

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```

- Enhanced `New-vRADeployment` with a new switch to define a custom version of vRealize Automation to deploy.
- Enhanced `Export-WSAJsonSpec` with a new switch to define a custom version of Workspace ONE Access to deploy.
- Enhanced `New-WSADeployment` with a new switch to define a custom version of Workspace ONE Access to deploy.

> - Enhanced `Set-NsxtManagerAuthPolicy` to handle password complexity parameters for NSX4.x.

Reviewer: ```suggestion
- Enhanced `Set-NsxtManagerAuthPolicy` to handle password complexity parameters for NSX 4.x.
```
